tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/AmqpClientTestSupport.java

```diff
@@ -218,7 +230,7 @@ protected TransportConfiguration addAcceptorConfiguration(ActiveMQServer server,
    }
 
    protected String getConfiguredProtocols() {
-      return "AMQP,OPENWIRE";
+      return "AMQP,OPENWIRE,CORE";
```

Comment:
This seems like the wrong place for this change. If a specific test wants to enable CORE it can override the method, thats what it exists for. (Same applies for the OPENWIRE one already there, but another time..)

Reply:
I agree with Robbie here.

Reply:
Will fix.